src/stories/containers/RecognizedDelegates/KeyStats/KeyDescriptionCards/MedianAnnualDai.tsx

```diff
@@ -0,0 +1,38 @@
+import styled from '@emotion/styled';
+import React from 'react';
+import GenericDelegateCard from '../../GenericDelegateCard';
+import { DescriptionDelegates } from './TotalRecognizedDelegatesCard';
+
+interface Props {
+  annual: string;
+}
+
+const MedianAnnualDai: React.FC<Props> = ({ annual }) => (
+  <ExtendedGenericDelegate>
+    <Annual>{`${annual} dai`}</Annual>
```

Comment:
Please, note that `annual` is going to be a number and should be formatted correctly, there is a `usLocalizedNumber` function that does the job in the `humanization.ts` file

Reply:
nice, done